features/desktop/handled_errors.feature

```diff
@@ -32,6 +32,14 @@ Feature: Handled Errors and Exceptions
         And the first significant stack frame methods and files should match:
             | Main.DoNotify()           |
 
+    Scenario: Custom App Type
```

Comment:
Projecting forwards, I presume you will have lots of custom config tests that you'll want to run.  Why not just call this "Custom Config" and add fields to it over time?   Then you can check them all in one go and save a load of test run time.

Reply:
(or you could leave it as-is and modify for the next field that's added)

Reply:
Great idea, will change it for the next one 👍